client-app/pages/account/order-details.vue

```diff
@@ -172,7 +172,9 @@ const breadcrumbs = useBreadcrumbs(() => [
 const isMobile = breakpoints.smaller("lg");
 const loadingAddItemsToCart = ref(false);
 
-const showPaymentButton = computed<boolean>(() => !!order.value && order.value.status === "New");
+const showPaymentButton = computed<boolean>(
```

Comment:
dont need <boolean> - TS gets it from the return 
![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/19205780/966c96cf-6c4a-4d99-8108-ce237c32dc38)